joint_limits_interface/include/joint_limits_interface/joint_limits_interface.hpp

```diff
@@ -54,6 +54,10 @@
 #include "joint_limits_interface/joint_limits_interface_exception.hpp"
 
 
+#define DEFAULT_POSITION_HANDLE {std::string(), "position"}
+#define DEFAULT_VELOCITY_HANDLE {std::string(), "velocity"}
+#define DEFAULT_COMMAND_HANDLE {std::string(), "position_command"}
```

Comment:
From what I understood, these macros are used to "default initialize" a joint handle. Wouldn't it make sense to add a new constructor to `Handle`? That way, the semantics are a bit clearer IMO.

Reply:
Yes, I thought of that and I do agree. I was only trying to limit my affect outside the joint limits module. Should the new constructor be a default (no args) or at least have the interface_name argument? Possibly 2 new constructors? Since it is now normal to use value semantics on Handle<> it should have a default no-args constructor, but I can see how setting interface_name is appropriate where possible...but in my case here it isnt necessary.

Reply:
A default constructor doesn't make much sense I believe - I mean what does that semantically imply? I can see however that a second constructor with a single interface name argument make sense. That's to say that there's a handle pointing to an individual joint, but the memory for it is being initialized to a later moment.

Reply:
>  I was only trying to limit my affect outside the joint limits module.

That's a fair point and I leave it up to you whether you want to continue. If you prefer leaving your PR as is, I would kindly ask you to open a follow-up issue for the changes on the handle class.

Reply:
ok, I will make the adjustments to Handle<> to add the 1-arg constructor. For the data initializer member function we could overload the set_value(double) to also have set_value(std::string joint_name, double value)...or just add the set_name() member?

Reply:
This sounds good for now. If we really crave a `set_name`, we can add when the time comes